tests/garage/experiment/test_resume.py

```diff
@@ -25,6 +26,8 @@ def teardown_method(self):
         super().teardown_method()
 
     def test_resume(self):
+        self.sess = tf.compat.v1.Session(graph=tf.Graph())
```

Comment:
shouldn't this be handled by TfGraphTestCase? I think this is our most buggy fixture, TBH.

Reply:
This is a hot fix - The test tried to initialize the same set of parameters twice in the same graph and restore the weights, and at the second time the parameters' name are all appended with `_1`, causing the restoring to fail. This fix essentially creates a new graph when restoring. I imagine in a real use case we won't really save and restore the test with the same session? even so, it's doable with this fix.